app/controllers/guides_controller.rb

```diff
@@ -2,9 +2,9 @@ class GuidesController < ApplicationController
   def index
     @user_options = User.pluck(:name, :id)
     @state_options = %w(draft published review_requested approved).map { |s| [s.titleize, s] }
-    @content_owner_options = ContentOwner.pluck(:title, :id)
 
-    @guides = Guide.includes(latest_edition: [:user, :content_owner])
+    # TODO: :content_owner not being included is resulting in an N+1 query
```

Comment:
I had a quick crack but it's not a quick fix. The problem is that because the :content_owner is a self referential join when searching we add a condition on the `tsv` column which is ambiguous because it's hand crafted SQL.

I'm up for fixing too but would prefer a separate PR?